__tests__/app.test.js

```diff
@@ -50,3 +50,44 @@ describe("/api/topics - Errors", () => {
       });
   });
 });
+//The created_at value was different, I just manually changed it.. feel like there is another way to do it
+describe("/api/articles/:article_id", () => {
+  it("GET 200: should respond with an article object", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toBeInstanceOf(Object);
+        const input = expect(body).toEqual({
```

Comment:
is there a need to assign this to a variable?

Reply:
Think I did this by accident and didn't notice